src/main/java/ukecat/Storage.java

```diff
@@ -0,0 +1,207 @@
+package ukecat;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+
+
+/**
+ * The Storage class manages the storage of tasks and related information for the UkeCat application.
+ * It includes methods for adding tasks and accessing the list of tasks.
+ */
+public class Storage {
+    // ukecat.Storage from input
+    public static String input; // user input
+    public static String[] words; // split words using
+    public static String desc; // for any task
+    public static LocalDate by; // for deadlines
+    public static LocalDate start; // for events
+    public static LocalDate end; // for events
+
+    public static int numT = 0;
```

Comment:
numT could be changed to a better and more representative name. perhaps numOfTasks?